src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
+  const cost = days * 40;
+  const discountDayThree = 3;
+  const discountDaySeven = 7;
```

Comment:
```suggestion
  const smallDiscount = 3;
  const largeDiscount = 7;
```
Погана практика - прив'язувати назви змінни до їх значень. А якщо термін для малої знижки колись поміняється і буде не 3 дні, а 4? Тоді назва буде взагалі неправильна